docs/UserGuide.md

```diff
@@ -5,48 +5,51 @@ title: User Guide
 * Table of Contents
 {:toc}
 --------------------------------------------------------------------------------------------------------------------
-
-## Introduction
- OneShelf is a desktop application for you to manage restaurant inventories and pending deliveries.
-It is easy to build and customise your inventories by using
-only Command Line Interface. If you are a busy restaurant manager
+## 1. Introduction
+OneShelf is a desktop application for you to manage restaurant inventories and pending deliveries.
+It is easy to build and customise your inventories and deliveries by using
+only Command Line Interface. If you are a fast-typist restaurant manager
 who prefers to use the Command Line Interface and needs to keep
 track of inventory items and pending deliveries, OneShelf is for you!
+<br><br>
+This User Guide document will provide a general overview of installation, existing features, coming-up features
+as well as summary commands.
+
 
 
-## Quick start
+--------------------------------------------------------------------------------------------------------------------
+## 2. Quick start
 
-1. Ensure you have Java `11` or above installed in your Computer.
+1. Ensure you have Java `11` or above irnstalled in your Computer.
 
 2. Download the latest `oneshelf.jar` from [here](https://github.com/AY2021S1-CS2103T-T12-1/tp/releases).
 
 3. Copy the file to the folder you want to use as the _home folder_ for your Inventory Book.
 
 4. Double-click the file to start the app. The similar GUI shown below (Figure 1) should appear in a few seconds. <br>
    *Note how the app contains some sample data but the installed version on your desktop might have a different data set.*<br>
-   ![Ui](images/Ui.png)
+   <br> ![Ui](images/Ui.png) <br>
    Figure 1: Introduction to OneShelf User Interface
 
 5. Type the command `help start` in the command box and press Enter to execute it. <br>
    A new help window shown below (Figure 2) should appear on your desktop.
-   ![Help Window](images/UiHelpStart.png)
+   <br> ![Help Window](images/UiHelpStart.png) <br>
    Figure 2: Help Start Window
    
 
-1. Refer to the [Features](#features) below for details of each command.
+6. Refer to the [Features](#features) below for details of each command.
 
 --------------------------------------------------------------------------------------------------------------------
-
-## Features
-
-<div markdown="block" class="alert alert-info">
+## 3. Features
 
 OneShelf has 2 main features that it is able to store, namely are:
 1. Inventory items
 2. Pending deliveries
 
 From here onwards, the term `item` and `delivery` are used specifically for inventory items and pending deliveries respectively.
+*You may want to refer to Figure 1 above.*
 
+<div markdown="block" class="alert alert-info">
 **:information_source: Notes about the item command format:**<br>
 
```

Comment:
For some reason, the **bold** markdown isn't applying in the preview

Reply:
![image](https://user-images.githubusercontent.com/46394439/97077689-46ee9880-1618-11eb-9b61-dc4f9d4b68ab.png)
seems to be fine on my side

Reply:
hmm I didn't check it on the website, but on github itself it looked different

Reply:
Like this

![image](https://user-images.githubusercontent.com/50513077/97078055-ad28ea80-161b-11eb-9cd8-6288859902b9.png)